.github/workflows/nightly-mjx-test-unit.yaml

```diff
@@ -0,0 +1,137 @@
+name: Nightly MJX unit test
```

Comment:
Please fold this into the build workflow. One build+test workflow should suffice.